packages/stage-ui/src/components/Misc/Alert.vue

```diff
@@ -31,20 +43,26 @@ const titleClass = computed(() => {
       return 'text-red-500 dark:text-red-400'
     case 'warning':
       return 'text-amber-500 dark:text-amber-400'
+    case 'success':
+      return 'text-green-500 dark:text-green-400'
+    case 'info':
+      return 'text-blue-500 dark:text-blue-400'
+    case 'loading':
+      return 'text-blue-500 dark:text-blue-400'
   }
   return ''
 })
 </script>
 
 <template>
-  <div class="flex flex-col gap-3 rounded-xl px-2 pb-3 pt-2" :class="containerClass">
+  <div class="flex flex-col gap-3 rounded-xl p-2" :class="containerClass">
     <div class="flex items-center gap-1.5 font-medium">
       <div class="text-2xl" :class="iconClass" />
       <div :class="titleClass">
         <slot name="title" />
       </div>
     </div>
-    <div class="px-1 text-sm">
+    <div v-if="$slots.content" class="px-1 text-sm">
```

Comment:
Can we avoid to use `$slots` / `$attrs` and `$props`? Try with useSlots or (i forgot the name)

```suggestion
    <div v-if="$slots.content" class="px-1 text-sm">
```